client/tests/integration/triggers/by_call_trigger.rs

```diff
@@ -337,7 +337,8 @@ fn trigger_in_genesis_using_base64() -> Result<()> {
 
     info!("WASM size is {} bytes", wasm.len());
 
-    let wasm_base64 = serde_json::json!(base64::encode(&wasm)).to_string();
+    let engine = base64::engine::general_purpose::STANDARD;
+    let wasm_base64 = serde_json::json!(base64::engine::Engine::encode(&engine, wasm)).to_string();
```

Comment:
Oh, why does simple `base64` encoding it should be that awkward now :(

Just complaining about API design of the crate, no objections agains this PR

Reply:
yeah, idk. I'm not base64 expert :)

Reply:
It's a weird thing that happened recently.. There are some recommendations with a bit more normal API https://twitter.com/mycoliza/status/1708933310548935148

Reply:
Time to have more posts from *tech* twitter in my feed